equinix/resource_network_device.go

```diff
@@ -1237,6 +1258,7 @@ func updateNetworkDeviceResource(primary *ne.Device, secondary *ne.Device, d *sc
 			secondary.LicenseFile = secondaryFromSchema.LicenseFile
 			secondary.LicenseToken = secondaryFromSchema.LicenseToken
 			secondary.CloudInitFileID = secondaryFromSchema.CloudInitFileID
+			secondary.ProjectID = secondaryFromSchema.ProjectID
```

Comment:
Same here, we don't need to populate it from schema.

Reply:
addressed